chia/consensus/condition_costs.py

```diff
@@ -11,5 +11,7 @@ class ConditionCost(Enum):
     ASSERT_HEIGHT_NOW_EXCEEDS = 0
     ASSERT_HEIGHT_AGE_EXCEEDS = 0
     RESERVE_FEE = 0
-    CREATE_ANNOUNCEMENT = 0
-    ASSERT_ANNOUNCEMENT = 0
+    CREATE_ANNOUNCEMENT_WITH_ID = 0
+    ASSERT_COIN_ANNOUNCEMENT = 0
```

Comment:
The two names above aren't really consistent (`COIN` vs `ID`)... consider renaming one of them.

Reply:
Maybe `CREATE_COIN_ANNOUNCEMENT` and `ASSERT_COIN_ANNOUNCEMENT`? This pair of names has a clear connection.